src/uu/dd/src/parseargs.rs

```diff
@@ -776,6 +776,42 @@ pub fn parse_seek_amt(
     }
 }
 
+/// Parse the amount of the input file to seek.
```

Comment:
`parse_seek_amt()`, `parse_skip_amt()`, `parse_iseek_amt()`, and `parse_oseek_amt()` are nearly identical. Can we refactor them to reduce code duplication in this part of the code?

Reply:
Combining these 4 requires the caller to pass in `*flags.seek_bytes`, since the input and output parsers take a different flags struct. If you think this is too aggressive of a change, I can combine them into 2 functions instead, keeping the signature the same aside from the new option parameter.